handbook/engineering/observability/query.md

```diff
@@ -0,0 +1,15 @@
+# Manual searching developer guide
```

Comment:
There is already some stuff about finding Prometheus metrics in https://about.sourcegraph.com/handbook/engineering/observability/monitoring#prometheus-and-alertmanager which you linked to - I would personally strongly prefer trying to keep the monitoring stuff in one place, and just have this guide there, so it can be maintained in one place.

For example, we have ["Adding monitoring"](https://about.sourcegraph.com/handbook/engineering/observability/monitoring#adding-monitoring), maybe we can have a "Finding monitoring" section? 😄

Reply:
Good call, thanks!